ivy/functional/ivy/layers.py

```diff
@@ -611,8 +611,12 @@ def multi_head_attention(
 
     With :class:`ivy.Container` input:
 
-    >>> x = ivy.Container(a=ivy.array([[[0.2, 1.1], [2.2, 3.4], [4.4, 5.6]]]),
-    ...                   b=ivy.array([[[1.4, 0.3], [1.2, 3.9], [0.4, 3.7]]]))
+    >>> x = ivy.Container(a=ivy.array([[[0.2, 1.1],
```

Comment:
Can you please explain why you think the existing docstring need to be reformated as they're rendering well from the [documentation](https://lets-unify.ai/ivy/functional/ivy/layers/multi_head_attention/multi_head_attention_functional.html)

Reply:
Because I found this Example "With a mix of ivy.Array and ivy.Container inputs:"
*ivy.Container* Like I did, so I want to make all examples the same show so I changed it.

Reply:
As Ivy supports broadcasting there is no problem in mixing both `ivy.Container` and `ivy.Array` also with this modification am pretty sure that the docs will not render properly from the docs. So I find it not necessary to modify anything in this function docstrings🙂.
The best way to go is to find an open task if you want to reformat a function and work on it.
How do you feel about it😊

Reply:
I got it. :)
So we will close this PR. Because there is nothing to do with `multi_head_attention` function because everything is good in this function.

Reply:
Yes, I see it that way feel free to go and look at open tasks and allocate one yourself to work on.
Best🙂